ngraph_bridge/ngraph_builder.cc

```diff
@@ -2878,13 +2878,13 @@ Status Builder::TranslateGraph(
   //
   {
     ngraph::pass::Manager passes;
-    passes.register_pass<pass::TransposeSinking>();
     if (GetEnv("TF_OV_CONSTANT_FOLDING") == "1") {
       passes.register_pass<ngraph::pass::ConstantFolding>();
     }
     if (GetEnv("TF_OV_TRANSPOSE_SINKING") != "0") {
-      passes.run_passes(ng_function);
+      passes.register_pass<pass::TransposeSinking>();
```

Comment:
Will TS be enabled by default, even if the user does not pass any env var?

Reply:
I think so. It will be disabled only when the user sets `TF_OV_TRANSPOSE_SINKING=0`.